pyvista/plotting/helpers.py

```diff
@@ -462,3 +462,52 @@ def plot_itk(mesh, color=None, scalars=None, opacity=1.0, smooth_shading=False):
     else:
         pl.add_mesh(mesh, color, scalars, opacity, smooth_shading)
     return pl.show()
+
+
+def view_vectors(view: str, negative: bool = False) -> Tuple[np.ndarray, np.ndarray]:
+    """Given a plane to view, return vectors for setting up camera.
+
+    Parameters
+    ----------
+    view : {'xy', 'yx', 'xz', 'zx', 'yz', 'zy'}
+        Plane to return vectors for.
+
+    negative : bool
+        Whether to view from opposite direction.  Default ``False``.
+
+    Returns
+    -------
+    numpy.ndarray
+        [x, y, z] vector that points in the viewing direction
+
+    numpy.ndarray
+        [x, y, z] vector that points to the viewup direction
```

Comment:
Where there are multiple returns, I think we're good to return the variable names (especially that they're useful). Agree with your points in https://github.com/pyvista/pyvista/issues/3422.

```suggestion
    vec : numpy.ndarray
        [x, y, z] vector that points in the viewing direction

    viewup : numpy.ndarray
        [x, y, z] vector that points to the viewup direction
```